src/libraries/System.Text.Json/tests/System.Text.Json.Tests/System.Text.Json.Tests.csproj

```diff
@@ -189,6 +189,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="..\..\src\System\Text\Json\BitStack.cs" Link="BitStack.cs" />
+    <Compile Include="Utf8JsonWriterTests.WriteRaw.cs" />
```

Comment:
nit: move this to the other item group above (line 188) with the rest of the utf8jsonwritertests files.